support-frontend/assets/helpers/flashSale.js

```diff
@@ -34,7 +34,7 @@ type SaleDetails = {
     price: number,
     annualPrice?: number,
     discountPercentage?: number,
-    saleCopy: SaleCopy,
+    saleCopy?: SaleCopy,
```

Comment:
The DP page no longer uses copy from the `Sale` object so I'm making this optional